packages/api/src/command/medical/tcm-encounter/__tests__/upsert-tcm-encounter.test.ts

```diff
@@ -0,0 +1,75 @@
+import { faker } from "@faker-js/faker";
+import { upsertTcmEncounter } from "../upsert-tcm-encounter";
+import { mockStartTransaction } from "../../../../models/__tests__/transaction";
+import { makeTcmEncounterModel } from "../../../../models/medical/__tests__/tcm-encounter";
+import { TcmEncounterModel } from "../../../../models/medical/tcm-encounter";
+
+jest.mock("../../../../models/medical/tcm-encounter");
+
+describe("Upsert TCM Encounter Command", () => {
+  beforeEach(() => {
+    jest.clearAllMocks();
+    jest.restoreAllMocks();
+    mockStartTransaction();
+  });
```

Comment:
We should clean up after ourselves on the test suites, so can we call `afterAll()` w/ `jest.restoreAllMocks();`, please?